Tests/EmbeddedScripts.JS.Jint.Tests/JintCodeRunnerTests.cs

```diff
@@ -0,0 +1,112 @@
+using System;
+using System.Threading.Tasks;
+using HelperObjects;
+using Jint.Runtime;
```

Comment:
Why do you need direct ref to Jint here? Ideally everything should be under your custom abstractions (custom Options, custom Exceptions, custom return value handling etc)

Reply:
Engine throws its specific exceptions on runtime or parser errors in js code, and I want to test do runner throws them next or not. 
> Ideally everything should be under your custom abstractions (custom Options, custom Exceptions, custom return value handling etc)

I added this question to tasks for grooming list at last Thursday.